.github/workflows/test_release_version.yml

```diff
@@ -1,53 +1,53 @@
-# name: "Test Release version"
-
-# on:
-#   push:
-#     branches:
-#       - 'release/**'
-
-# env:
-#   OTP_VERSION: 23.3
-#   ELIXIR_VERSION: 1.11.4
-#   PHOENIX_VERSION: 1.5.7
-#   MIX_ENV: test
-
-# jobs:
-#   release_version_test:
-#     runs-on: ubuntu-latest
-
-#     steps:
-#       - name: Cancel Previous Runs
-#         uses: styfle/cancel-workflow-action@0.8.0
-#         with:
-#           access_token: ${{ github.token }}
-
-#       - uses: actions/checkout@v2
-#         with:
-#           ref: ${{ github.head_ref }}
-
-#      - uses: erlef/setup-elixir@v1
-#        with:
-#          otp-version: ${{ env.OTP_VERSION }}
-#          elixir-version: ${{ env.ELIXIR_VERSION }}
-
-#       - name: Cache Elixir build
-#         uses: actions/cache@v2
-#         with:
-#           path: |
-#             _build
-#             deps
-#           key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
-#           restore-keys: |
-#             ${{ runner.os }}-mix-
-
-#       - name: Install Dependencies
-#         run: mix deps.get
+name: "Test Release version"
+
+on:
+  push:
+    branches:
+      - 'release/**'
+
+env:
+  OTP_VERSION: 23.3
+  ELIXIR_VERSION: 1.11.4
+  PHOENIX_VERSION: 1.5.7
+  MIX_ENV: test
+
+jobs:
+  release_version_test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.8.0
```

Comment:
The latest version is [0.9.0](https://github.com/styfle/cancel-workflow-action/releases). We can consider using it.

Reply:
I think it better to upgrade on another PR along with other workflow as well @longnd